### PR TITLE
Create Home Section and Vision Section

### PR DESCRIPTION
**Pull request by mr-vijaychauhan**
### 🛠️ Fixes Issue

Create Home Section & Vision Section  with responsive website and also create Component for Title with Arrow `(Title.js`).

### 👨‍💻 Changes proposed
 Create Home Section & vision section for `Index.js`

### 🔧 Check List (Check the type of issue it fixes) 

- [ ] Bug Fix
- [ ] Add Home & Vision Section
- [ ] Enhancement
- [ ] Create Components (`Title.js`)

### ✔️ Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have done `git pull` before adding my new changes
- [ ] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers
I am stuck here. when I change width of grid, its not according to this design otherwise fine. I cannot fix this issue because i'm expert in design.

Desktok-GDSC

### Desktop View
Desktok-GDSC

### Responsive View
Responsive-GDSC

### CODE REVIEW

File: components/Index.js [hunk 1]
```diff
@@ -1,47 +1,25 @@
...
+            Welcome to VIT Chennai's Google Developer Student Club!
+          </h1>
+          <p className="py-2 text-2xl max-w-xl my-2">
+            At GDSC VITC we aim to help each other learn and grow. This is where
+            coders,designers and managers come together to create and inspire
+            many. Join us to be a part of this exciting adventure!
+          </p>
+        </div>
+        <div className="px-4 py-8">
+          <Image src={HeroImage} width={912} height={912} />
```
[the remark this answers is not in the document]
Comment (Arjun31415) [reply components/Index.js:20]: Please give an `alt` prop

File: components/Title.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+import DownArrow from "../Assets/Icons/down-arrow.png";
+
+const Title = ({ title }) => {
+  return (
+    <div className="flex flex-col items-center justify-center ">
+      <Image src={DownArrow} width={30} height={30} className="pb-5" />
```
[the remark this answers is not in the document]
Comment (Arjun31415) [reply components/Title.js:7]: Similarly here

**Review by Arjun31415 — CHANGES_REQUESTED** [review]
Check the images and give an alt property

**Review by Arjun31415 — APPROVED** [review]

**Comment by mr-vijaychauhan** [comment]
Updated alt Prop for Image. can you check ??

**Comment by FidalMathew** [comment]
@mr-vijaychauhan when we view the website on mobile screens, can you change the view of the Navbar?

Current behavior: Open navbar
Expected behavior: Closed navbar with hamburger icon and onlick -> view expanded navbar

**Comment by mr-vijaychauhan** [comment]
@FidalMathew  I will do responsive navbar but you accept this pull request then I create issue for this responsive navbar becuase i have work so no time. i will do later after sometimes.

**Comment by FidalMathew** [comment]
@mr-vijaychauhan let's do one thing, send the PR to the "dev" branch.

**Comment by mr-vijaychauhan** [comment]
@FidalMathew change branch main to dev for this pull request.

**Pull request merged**